_posts/2022-08-24-kubeflow-1.6-release.md

```diff
@@ -0,0 +1,143 @@
+ the central dashboard and enables the PodDefaults webhook to pick-up new certificate updates.   For the Kubernetes upgrade,
+ the community developed and tested each Kubeflow component manifest with Kubernetes v1.22.  As v1.22 introduced some breaking changes, the upgrade was a team effort, and this Kubeflow release management process will be useful in the community's future updates of Kubernetes and other software dependencies. The software is available [here](https://github.com/kubeflow/kubeflow/releases/tag/v1.6.0). 
+ 
+In parallel to developing these new software features, the Kubeflow Community also completed its annual user survey.
+ The survey generated many good user insights into requirements for Kubeflow, and you can see more on the
+ [survey results](https://docs.google.com/document/d/1TLl8iFWQ-x2pVAHV9G8GFYMgAOuN2VFOnZV2JQNA-uc/edit#). Of the many
+ highlights, we would like to identify the growing request for model monitoring as shown in the chart below.
+
+![2022 User Survey Gaps In ML Activities and Workflow](/images/2022-08-24-kubeflow-1.6-release/2022-user-survey-gaps-in-ml-activities-and-workflow.png)
+
+We believe the increased focus on monitoring identifies Kubeflow’s maturing user base. The Community is making efforts
+ to explain Kubeflow’s current functionality and build into the users’ new model monitoring requirements. In addition,
+ the user survey results and Kubeflow Working Group roadmaps will be discussed in the upcoming Kubeflow Summit. You can
+ learn more about the Kubeflow Summit [here](https://docs.google.com/document/d/1fLg1EqVtJekcXxT8GM_Rqg0-G-vIDVCrzuxE06Oq9dI/edit).
+
+To round back to the current software delivery, the following takes a deeper look at the v1.6 highlights in more detail.
+
+## Kubeflow v1.6 details
+
+From a new feature introduction standpoint, Kubeflow v1.6 includes an experimental release of the KFP v2’s new front-end,
+ back-end and SDK, which deliver a modern UI and DAG, first class support for metadata, and a simplified component
+ authoring experience. This slide deck, [KFP v2 Introduction](https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gecbd775238_0_20),
+ provides a good overview of v2. 
+
+<br />
+
+The KFP v2 alpha introduces: 
+
+- An Argo-agnostic approach for creating and running pipelines 
+- A brand new DAG visualization, which uses the Pipeline Template and MLMD in this [pipeline spec](https://github.com/kubeflow/pipelines/blob/master/api/v2alpha1/pipeline_spec.proto)
+- [Streamlined Component Authoring](https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gecbd775238_0_438)
+
+<br />
+
+You can learn more about KFP v2 and the related breaking changes in these docs:
+
+- [KFP SDK V2 CLI design](ahttps://docs.google.com/document/d/1Z79ebKaf3hjyadSRsme1Qo7F-Tqu4qM1cyKuLeu2PgA/edit)
```

Comment:
I think the slide "Understanding KFP v2" [1] is still a good overall reference for learning more about KFP v2.
In comparison, "KFP SDK V2 CLI design" is not essential to KFP v2, and can be skipped in this context. 

[1] https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gb4a3fac3a8_7_1911

Other than this, LGTM. Thanks!

Reply:
Replaced the KFP SDK v2 CLI design link with the KFP v2 introduction link

Reply:
@annajung lines 41 and lines 56 both refer to users to the KFP v2 Introduction slide deck.   We might delete line 56.